docs/source/quickstart.rst

```diff
@@ -681,6 +681,49 @@ service.
             }
         }
 
+Building the Model
+==================
+
+Now that we have a simple model we'll want to build it. Building the model will
```

Comment:
minor: The language in the paragraph around "we" and "us" seems weird to me. Seems like it could be more like 

> Now that you have a model, you'll want to build it an generate things from it. Building the model creates projections of the model, applies plugins to generate artifacts, and creates a JAR that contains the filtered projection. The `Smithy Gradle Plugin`_ is the best way to get started building a Smithy model. First, create a ``smithy-build.json`` file: